doc/source/user_guide/index.rst

```diff
@@ -89,20 +98,23 @@ additional interface features that are not possible via the ``tui`` object:
 
     energy_is_enabled = root.setup.models.energy.enabled()
 
-For more information, see :ref:`ref_settings`. For the full hierarchy under the ``root``
-object, see the API topic :ref:`ref_solver_tui`.
-
-For general guidance on using both the  ``tui`` and ``root`` objects, see
-:ref:`ref_user_guide_solver_settings`.
+For the full hierarchy under the solver ``root`` object, see :ref:`ref_solver_tui`.
+For additional interface features, see :ref:`ref_settings`.
 
 Meshing mode session
 --------------------
-A meshing mode session has an active ``meshing`` object, which provides
-three interfaces to the mesher.
+A meshing mode session has an active ``meshing`` object that provides two
+distinct interfaces to the mesher:
```

Comment:
The original content said that there were three distinct interfaces to the mesher. However, based on the content in this section, it seems that "meshing" and workflow" properties work together to provide one interface to Fluent's meshing workflows. If I am misunderstanding, please revise this section.

Reply:
meshing is divided into tui on the one hand and four related objects on the other (meshing, workflow, PartManagement, PMFileManagement) which are inter-dependent, and can be referred to collectively as meshing workflow.